packages/card/src/Card.js

```diff
@@ -43,6 +53,22 @@ const Root = createStyledComponent(
 /**
  * Card component
  */
-export default function Card(props: Props) {
-  return <Root {...props} />;
+export default class Card extends Component {
+  props: Props;
+
+  render() {
+    const rootProps = {
+      onKeyPress: this.props.onClick ? this._onKeyPress.bind(this) : undefined,
+      tabIndex: this.props.onClick ? 0 : undefined,
+      ...this.props
+    };
+    return <Root {...rootProps} />;
+  }
+
+  _onKeyPress(event: Object) {
+    if (event.key === 'Enter' || event.key === ' ') {
```

Comment:
`event.key` is only recently supported in Safari - as of March 2017, though it is within the last 2 versions, so yay!

http://caniuse.com/#search=event.key